packages/desktop-client/src/components/filters/FiltersMenu.jsx

```diff
@@ -231,7 +233,7 @@ function ConfigureField({
               });
             }}
           >
-            Apply
+            <Trans>Apply</Trans>
```

Comment:
Not sure whether to prefer `<Trans>` if there is no tags or components to be included. It could also be written as `{t('Apply')}` instead?

Reply:
This was already merged. I guess if you have changes, open a PR?